zzz_generated/app-common/app-common/zap-generated/cluster-objects.h

```diff
@@ -7077,12 +7077,12 @@ enum class Fields
 struct Type
 {
 public:
-    uint16_t actionID;
+    uint16_t actionID = static_cast<uint16_t>(0);
     chip::CharSpan name;
-    ActionTypeEnum type;
-    uint16_t endpointListID;
-    uint16_t supportedCommands;
-    ActionStateEnum status;
+    ActionTypeEnum type        = static_cast<ActionTypeEnum>(0);
```

Comment:
ActionTypeEnum::kOther

Generally I think it is better for all enums to have readable values of defaults instead of just "whatever 0 is" and not actually know if 0 is even valid for the underlying type.

Reply:
There is no `kOther`.

The point here is not really to have "sane" defaults.  In particular, a default of 0 can easily be out of valid range for integers too.  The only way to have sane defaults is described in https://github.com/project-chip/connectedhomeip/pull/13432#discussion_r782509765

The point here is to at least not be reading un-initialized memory, so behavior is not random.